matter/wifi/wf200/efr_gpio.c

```diff
@@ -68,6 +68,6 @@ void sl_wfx_host_gpio_init(void)
 
   // Change GPIO interrupt priority (FreeRTOS asserts unless this is done here!)
   NVIC_ClearPendingIRQ(1 << wirq_irq_nb);
-  NVIC_SetPriority(GPIO_EVEN_IRQn, 5);
-  NVIC_SetPriority(GPIO_ODD_IRQn, 5);
+  NVIC_SetPriority(GPIO_EVEN_IRQn, NVIC_PRIORITY);
```

Comment:
had a similar comment before `WFX_SPI_NVIC_PRIORITY` would be nice

Reply:
changed